skfp/fingerprints/base.py

```diff
@@ -39,6 +39,10 @@ def __init__(
         self.verbose = verbose
         self.random_state = random_state
 
+        # value that helps to tell apart fingerprint and utility transformers
+        # set as True for util transformers
+        self.fingerprint_transformer = True
```

Comment:
We should just create separate parent classes. Or, better, just don't use parent classes at all for those utility transformers.